packages/flex-plugin-scripts/src/config/webpack.config.ts

```diff
@@ -236,12 +235,17 @@ export const _getPlugins = (env: Environment): Plugin[] => {
     __FPB_REACT_DOM_VERSION: `'${reactDOMVersion}'`,
   }));
 
-  if (env === Environment.Production) {
-    plugins.push(new SourceMapDevToolPlugin({
-      append: '\n//# sourceMappingURL=bundle.js.map',
-    }));
-  }
+  return plugins;
+};
 
+export const _getFlexPlugins = (env: Environment): Plugin[] => {
```

Comment:
Sorry, should have done these myself, but can you make sure every function has a docblock?

Reply:
Also rename to `_getStaticPlugins`